pallets/utility/src/lib.rs

```diff
@@ -799,6 +788,30 @@ pub mod pallet {
             Self::deposit_event(Self::run_batch(origin, is_root, calls, false));
             Ok(())
         }
+
+        /// Send a call through an indexed pseudonym of the sender.
+        ///
+        /// Filter from origin are passed along. The call will be dispatched with an origin which
+        /// use the same filter as the origin of this call.
+        ///
+        /// The dispatch origin for this call must be _Signed_.
+        #[pallet::call_index(9)]
+        #[pallet::weight({
+            let dispatch_info = call.get_dispatch_info();
+			(
+				<T as Config>::WeightInfo::as_derivative()
+					.saturating_add(T::DbWeight::get().reads_writes(1, 1))
```

Comment:
I think Substrate is wrong to include these extra storage reads/writes.

Reply:
The commend from the Substrate code is:
`// AccountData for inner call origin accountdata.`
But there shouldn't be a read/write of the `AcountData` (system pallet).  Since the derivative account doesn't pay the tx fees.  The caller `origin` pays.